mmv1/third_party/terraform/website/docs/r/apigee_api.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+subcategory: "Apigee"
+page_title: "Google: google_apigee_api"
+description: |-
+    An Apigee API proxy is essentially a layer that sits in front of your backend APIs. It acts as an intermediary between your API consumers (like mobile apps or websites) and your backend services.   
+
+    Think of it like a gatekeeper or a middleman:
+
+    * Decoupling: It decouples the app-facing API from your backend services. This means you can make changes to your backend systems without affecting the apps that use your API, as long as the API proxy interface remains consistent.   
+    * Abstraction: It hides the complexities of your backend systems, presenting a simplified and consistent interface to your API consumers.
+    * Control: It gives you fine-grained control over how your APIs are accessed and used, allowing you to enforce security policies, rate limits, and other controls.
+---
+
+# google_apigee_api
+
+To get more information about API proxies see, see:
+
+* [API documentation](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.apis)
+* How-to Guides
+  * [API proxies](https://cloud.google.com/apigee/docs/resources)
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` -
+  (Required)
+  The ID of the API proxy.
+
+* `org_id` -
+  (Required)
+  The Apigee Organization name associated with the Apigee instance.
+
+* `config_bundle` -
+  (Required)
+  Path to the config zip bundle.
+
+- - -
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `id` - an identifier for the resource with format `organizations/{{org_id}}/apis/{{name}}`
+
+* `meta_data` -
+  Metadata describing the API proxy.
+  Structure is [documented below](#nested_meta_data).
+
+* `revision` -
+  A list of revisions of this API proxy.
+
+* `latest_revision_id` -
+  The id of the most recently created revision for this API proxy.
+
+* `md5hash` -
+  (Computed) Base 64 MD5 hash of the uploaded data. It is speculative as remote does not return hash of the bundle. Remote changes are detected using returned last_modified timestamp.
+
+* `detect_md5hash` -
+  (Optional) Detect changes to local config bundle file or changes made outside of Terraform. MD5 hash of the data, encoded using base64. Hash is automatically computed without need for user input.
```

Comment:
I don't think this would be under `Attributes` if it allows the user to set it. If the user can't set it, it should be not marked as (Optional)

Reply:
Thay are part of the schema, so we should keep them